trends_presentation/src/commonTest/kotlin/net/thechance/mena/trends/presentation/screen/upload_reel/UploadTrendViewModelTest.kt

```diff
@@ -61,60 +58,67 @@ class UploadTrendViewModelTest: TestExtensions() {
     }
 
     @Test
-    fun `onRetrieveVideo should call validateDuration if size is valid`() = runTest(testDispatcher) {
-        viewModel.onRetrieveVideo(defaultFile, ::defaultReadBytes)
+    fun `onRetrieveVideo should call validateDuration if size is valid`() =
+        runTest(testDispatcher) {
+            viewModel.onRetrieveVideo(defaultFile, ::defaultReadBytes)
 
-        verify(exactly(1)) { validator.validateDuration(any()) }
-    }
+            verify(exactly(1)) { validator.validateDuration(any()) }
+        }
 
     @Test
-    fun `onRetrieveVideo should not call validateDuration if validation size throws exception`() = runTest(testDispatcher) {
-        viewModel.onRetrieveVideo(fileWithInvalidSize, ::defaultReadBytes)
+    fun `onRetrieveVideo should not call validateDuration if validation size throws exception`() =
+        runTest(testDispatcher) {
+            viewModel.onRetrieveVideo(fileWithInvalidSize, ::defaultReadBytes)
 
-        verify(exactly(0)) { validator.validateDuration(any()) }
-    }
+            verify(exactly(0)) { validator.validateDuration(any()) }
+        }
 
     @Test
-    fun `onRetrieveVideo should not call validateDuration if duration is null`() = runTest(testDispatcher) {
-        everySuspend { videoExtractor.getDuration(any()) } returns null
+    fun `onRetrieveVideo should not call validateDuration if duration is null`() =
+        runTest(testDispatcher) {
+            everySuspend { videoExtractor.getDuration(any()) } returns null
 
-        viewModel
-        viewModel.onRetrieveVideo(fileWithInvalidSize, ::defaultReadBytes)
+            viewModel
+            viewModel.onRetrieveVideo(fileWithInvalidSize, ::defaultReadBytes)
 
-        verify(exactly(0)) { validator.validateDuration(any()) }
-    }
+            verify(exactly(0)) { validator.validateDuration(any()) }
+        }
 
     @Test
-    fun `onRetrieveVideo should update error state with FileTooLarge if size is not valid`() = runTest(testDispatcher) {
-        viewModel.onRetrieveVideo(fileWithInvalidSize, ::defaultReadBytes)
-        advanceUntilIdle()
-
-        viewModel.state.test {
-            assertThat(awaitItem().errorState).isEqualTo(ErrorState.FileTooLarge)
-            cancelAndIgnoreRemainingEvents()
+    fun `onRetrieveVideo should update error state with FileTooLarge if size is not valid`() =
```

Comment:
I think the keyword( is not valid) contains different meanings , but in this case you test only when size of file exceed the max limit we determined so I suggest rename the test to be : 
```suggestion
    fun `onRetrieveVideo should update error state with FileTooLarge when file's size exceed the max limit`() =
```

Reply:
i didn't write this test case i just reformat the entire file, but okay